src/Functions/date_trunc.cpp

```diff
@@ -0,0 +1,170 @@
+#include <Columns/ColumnConst.h>
```

Comment:
fyi, new files need to be added to `ya.make` using `utils/generate-ya-make/generate-ya-make.sh`

Reply:
thank you - I don't know about it.
fixed.